spec/system/user/view_user_index_spec.rb

```diff
@@ -63,9 +63,9 @@ def shows_comments
 
       def shows_comment_timestamp
         within("#substories .profile-comment-card .profile-comment-row:first-of-type") do
-          ts = comment.decorate.published_timestamp
-          timestamp_selector = ".comment-date time[datetime='#{ts}']"
-          expect(page).to have_selector(timestamp_selector)
+          iso8601_date_time = /^((\d{4})-(\d{2})-(\d{2})T(\d{2}):(\d{2}):(\d{2})Z)$/
```

Comment:
Not being too strict here, but just verifying that we're including the right number of digits in the right order.

![image](https://user-images.githubusercontent.com/11466782/109323276-e87d3500-7818-11eb-8afe-de62e859a8c8.png)